frontend/src/Components/ProviderHomePage/ProviderHomePage.jsx

```diff
@@ -53,19 +51,6 @@ const ProviderHomePage = () => {
         }
     };
 
-    const fetchReviews = async () => {
```

Comment:
Are we removing this because of the update in user stories?
Sorry for the late update : ( if you already wrote it it’s fine to add it back in later! You just don’t have to do additional work on top of this